src/NuGetRepack/Program.cs

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Roslyn.Tools.NuGet.Repack
+{
+    internal sealed class Program
+    {
+        private enum Operation
+        {
+            None = 0,
+            Release = 1,
+            PreRelease = 2,
+        }
+
+        private static int Main(string[] args)
+        {
+            var operation = Operation.None;
+            var packages = new List<string>();
+            string outDirectory = null;
+
+            try
+            {
+                int i = 0;
+                while (i < args.Length)
```

Comment:
Consider using Mono.options for the command line arguments. I've been using it for a lot of tools recently. Really simplifies the parsing and printing of help messages.

Reply:
I don't think there is any benefit of adding extra dependency on Mono.Options. This is very trivial code. BTW, Mono.Options is not signed which is causing problems.